packages/axes/src/components/Axes.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react'
+import { Axis } from './Axis'
+import { AnyScale, AxisProp } from '../types'
+
+const positions = ['top', 'right', 'bottom', 'left'] as const
```

Comment:
Positions are also defined in the canvas implementation, maybe we can move this to a types file.